ldms/src/sampler/lustre_mdt/lustre_mdt_general.c

```diff
@@ -151,8 +151,15 @@ char *mdt_general_osd_path_find(const char * const *paths, const char *component
                 }
 
         }
+        if (osd_dir != NULL) {
```

Comment:
This is good to revive the logging. Don't forget we need the same in lustre_ost_general.c